route_linux.go

```diff
@@ -60,6 +61,74 @@ func (n *NexthopInfo) ListFlags() []string {
 	return listFlags(n.Flags)
 }
 
+type MPLSDestination struct {
+	Labels []int
+}
+
+func (d *MPLSDestination) Family() int {
+	return nl.FAMILY_MPLS
+}
+
+func (d *MPLSDestination) Decode(buf []byte) error {
```

Comment:
seems odd to use serialize/decode. I think it should be encode/decode or serialize/deserialize here